### ISSUE TITLE
exoplayer srt tag support and hdr flag in view

### ISSUE THREAD

**Issue by satstarter** [jellyfin/jellyfin-androidtv#1918]
### Describe the feature you'd like

hi the hdr flag in view was asked a year ago
and srt tag support is something i realy want for myself )p
so if there's anny chanche someone of the coders could make this happen?

ps my coding is to poor to do something like this BUT i can help testing or if i can help
whit something else let me know

thanx in advance

**Comment by thornbill** [comment]
Seems like this is a duplicate of #1177?

**Comment by satstarter** [comment]
yes you're right also srt tag support seems duplicate of  #1804 
sorry

**Comment by thornbill** [comment]
No worries. I just wanted to be sure I was not missing something. 🙂
